lib/public/models/OverviewModel.js

```diff
@@ -35,12 +35,13 @@ import { buildUrl } from '../utilities/fetch/buildUrl.js';
 export class OverviewPageModel extends Observable {
     /**
      * Constructor
+     * @param {boolean} hasStableBeams - Flag indicating whether to fetch items with stable beams.
      */
-    constructor() {
+    constructor(hasStableBeams = false) {
```

Comment:
Stable Beam is way too specific to be on overview model. Please add it to specific models only. If needed, you can use composition to factorize some filterings.